run_docker_iqfeed.bat

```diff
@@ -0,0 +1,59 @@
+@echo off
+
+REM QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+REM Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+REM
+REM Licensed under the Apache License, Version 2.0 (the "License");
+REM you may not use this file except in compliance with the License.
+REM You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+REM
+REM Unless required by applicable law or agreed to in writing, software
+REM distributed under the License is distributed on an "AS IS" BASIS,
+REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+REM See the License for the specific language governing permissions and
+REM limitations under the License.
+
+set docker_params=--expose 5009^
```

Comment:
@jaredbroad not sure these are mandatory but at least now we have the ability the pass them in.

Reply:
Lets find out which ones we need for sure and scrap the rest

Reply:
guys, let's try without any, should be working fine as well. at least we gonna have the ability to add some parameters if needed like --net=host for those who require it.